updating/updating-restricted-network-cluster/restricted-network-update.adoc

```diff
@@ -6,6 +6,8 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+Use the following procedures to update a cluster in a disconnected environment without installing a local OpenShift Update Service.
```

Comment:
We should say `Use the following procedures to update a cluster in a disconnected environment if you do not have access to a OpenShift Update Service running in the disconnected environment`